content/2_Bronze/Intro_Sets.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 id: intro-sets
-title: Introduction to Sets & Maps
+title: (Optional) Introduction to Sets & Maps
```

Comment:
why did this become optional?

Reply:
benq said it should
and in my very crappy memory i don't think sets/maps have come up in bronze? it's all just arrays

Reply:
@bqi343 on second thought, i kinda feel like given how necessary sets & maps are in any later division, this shouldn't be marked as optional...

Reply:
it's optional for the division it's in

Reply:
i mean i think most people see "optional" and just skip it regardless
for me this word means that like "hey you _might_ need to know this, but no biggies if you don't wanna"

Reply:
https://usaco.guide/general/modules

> Some relatively rare modules are prefixed with "optional." It's not essential to work through these if your goal is just to pass the corresponding division. They might be referenced in later divisions.

Reply:
lowk unintuitive, but sure